src/index.js

```diff
@@ -354,7 +371,17 @@ export default class TruncateMarkup extends React.Component {
     if (!node) {
       return node;
     } else if (typeof node === 'string') {
-      return this.splitString(node, splitDirections);
+      if (this.policy.tokenizeString) {
```

Comment:
I think this whole block could be removed because we already have it inside the `splitString` method, and it seems that's the correct place to do it in.

Reply:
yes, you are right